framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/HACepScenarioImpl.java

```diff
@@ -0,0 +1,145 @@
+        final File amqStreamsZipFile = new File(buildDirectory, AMQ_STREAMS_ZIP);
+        try {
+            Request.Get(OpenShiftConstants.getAMQStreamsZip()).execute().saveContent(amqStreamsZipFile);
+        } catch (IOException e) {
+            throw new RuntimeException("Unable to download AMQ streams zip", e);
+        }
+
+        final File amqStreamsDirectory = new File(buildDirectory, AMQ_STREAMS_DIRECTORY);
+        amqStreamsDirectory.mkdir();
+        final Unzip unzip = new Unzip();
+        unzip.setSrc(amqStreamsZipFile);
+        unzip.setDest(amqStreamsDirectory);
+        unzip.execute();
+
+        return amqStreamsDirectory;
+    }
+
+    private static void filterNamespaceInInstallationFiles(final File amqStreamsInstallDirectory,
+                                                           final String projectName) {
+        final List<File> installationFiles = Arrays.asList(amqStreamsInstallDirectory.listFiles());
+        for (File file: installationFiles) {
+            if (file.isFile() && file.getName().contains("RoleBinding")) {
+                try {
+                    String fileContent = FileUtils.readFileToString(file, "UTF-8");
+                    fileContent = fileContent.replaceAll("namespace: .*",
+                                                         "namespace: " + projectName);
+                    FileUtils.write(file, fileContent, "UTF-8");
+                } catch (IOException e) {
+                    throw new RuntimeException("Unable to modify AMQ streams installation files");
+                }
+            }
+        }
+    }
+
+    private static List<String> sortedFolderContent(final File folder) {
+        return Arrays.asList(folder.listFiles())
+                .stream()
+                .sorted((f1, f2) -> f1.getName().compareTo(f2.getName()))
+                .map(f -> f.getAbsolutePath())
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public List<Deployment> getDeployments() {
+        final List<Deployment> deployments = new ArrayList<>();
+        deployments.add(new HACepDeploymentImpl(project));
+
+        return deployments;
+    }
+
+    private Properties loadFilteredProperties() {
+        final Properties filteredProperties = new Properties();
+        try (InputStream is = HACepScenarioImpl.class.getResourceAsStream(FILTERED_PROPERTIES)) {
+            filteredProperties.load(is);
+        } catch (IOException e) {
+            throw new RuntimeException("Unable to load AMQ streams zip: " + FILTERED_PROPERTIES, e);
+        }
+
+        return filteredProperties;
+    }
+
+    @Override
+    public File getAMQStreamsDirectory() {
+        final Properties filteredProperties = loadFilteredProperties();
+        final String buildDirectory = filteredProperties.getProperty(BUILD_DIRECTORY_PROPERTY);
+        final File amqStreamsDirectory = downloadAndUnzipAMQStreams(buildDirectory);
```

Comment:
Why do you download and unzip AMQ resources here? It is done already in deployKieDeployments().